src/Altinn.AccessManagement/Models/Rights/RightDelegationOfferedInput.cs

```diff
@@ -0,0 +1,29 @@
+using System.ComponentModel.DataAnnotations;
+using Altinn.AccessManagement.Utilities;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Altinn.AccessManagement.Models;
+
+/// <summary>
+/// Input model
+/// </summary>
+public record RightDelegationOfferedInput
```

Comment:
Naming should perhaps be more general, as this can be reused across all endpoints acting on behalf of an authorized party. AuthorizedPartyInput?